docs/docs/rest-api/public/api/v2/tasks.raml

```diff
@@ -0,0 +1,43 @@
+get:
+  description:
+    List all running tasks.
+  is: [ secured ]
+  queryParameters:
+    status:
+      description: Filter the list of tasks by status
+      enum: [ TASK_STAGING, TASK_STARTING, TASK_RUNNING, TASK_FINISHED, TASK_FAILED, TASK_KILLED, TASK_LOST, TASK_ERROR ]
```

Comment:
Not quite. According to the [old docs](https://mesosphere.github.io/marathon/docs/rest-api.html#get-v2-tasks) and [the implementation](https://github.com/mesosphere/marathon/blob/d098e5f73905d91e9511dbbdf970472656d4649b/src/main/scala/mesosphere/marathon/api/v2/TasksResource.scala#L135-L139), possible values are `[running,staging]`